src/main.rs

```diff
@@ -136,9 +135,10 @@ async fn main() -> miette::Result<()> {
         .unwrap();
 
     let cli = Cli::parse();
-    let credentials = Credentials::load().await?;
 
-    match cli.command {
+    // handle init separately, since that is the only command that does not assume an initialized
```

Comment:
I dislike this change, as stated earlier

Reply:
Please elaborate.

Reply:
https://github.com/helsing-ai/buffrs/pull/145/files#r1368292580

Reply:
Addressed in the linked comment.